src/pages/WorkflowInputs.js

```diff
@@ -1,28 +1,74 @@
 import { Fragment } from 'react'
 import { div, h, h3, span } from 'react-hyperscript-helpers'
 import { IdContainer } from 'src/components/common'
-import { TextArea } from 'src/components/input'
+import { TextArea, TextInput } from 'src/components/input'
 import { FormLabel } from 'src/libs/form'
 
 
-export const WorkflowInputs = ({ workflowUrl, workflowInputs, setWorkflowInputs }) => {
+export const WorkflowInputs = ({ workflowUrl, entityType, setEntityType, entityId, setEntityId, workflowInputsDefinition, setWorkflowInputsDefinition }) => {
+  // used as placeholder to let users know expected structure of inputs definition.
+  // To be removed later when we design UI for entering the input definition.
+  const inputMappingExample = '\n[\n' +
```

Comment:
Would it be easier to write this out in JSON and then use a `toString` method to generate the text box contents?

Reply:
Good point. I will try that.